clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -4946,6 +4946,12 @@ void CXXNameMangler::mangleExpression(const Expr *E, unsigned Arity,
     E = cast<ConstantExpr>(E)->getSubExpr();
     goto recurse;
 
+  case Expr::CXXReflectExprClass: {
+    // TODO(Reflection): implement this after introducing std::meta::info
+    // and add info in APValue
+    break;
```

Comment:
```suggestion
    [[fallthrough]];
```
We should error that we can’t mangle this yet until we implement support for it